notebook/static/tree/js/notebooklist.js

```diff
@@ -697,6 +707,73 @@ define([
         });
     };
 
+    NotebookList.prototype.move_selected = function() {
+        // TODO(nhdaly): Support moving multiple items at once.
+        if (this.selected.length !== 1){
+            return;
+        }
+
+        var that = this;
```

Comment:
I think it would make more sense to move this variable declaration to the top of the function and use `that` instead of `this` throughout.